public/client.js

```diff
@@ -10,40 +10,40 @@ $(function() {
     // Initialize varibles
     var $window = $(window);
     var $usernameInput = $('.usernameInput'); // Input for username
-    var $messages = $('.messages'); // Messages area
-    var $inputMessage = $('.inputMessage'); // Input message input box
 
-    var $loginPage = $('.login.page'); // The login page
-    var $chatPage = $('.chat.page'); // The chatroom page
+    var $loginPage = $('#loginPage'); // The login page
+    var $mapPage = $('#mapPage'); // The chatroom page
+    var $adminPanel = $('#adminPanel');
+    var $playerPanel = $('#playerPanel');
 
     // Prompt for setting a username
     var username;
     var connected = false;
-    var typing = false;
-    var lastTypingTime;
-    var $currentInput = $usernameInput.focus();
+
+    var adminName = "a";
```

Comment:
Use capitalisation (ADMIN_NAME) for constants. There is a const modifier in javascript but some browsers don't support it.